2020-06-15_15-52-19: DF Exception: Sub: R1015J, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1015J_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-20: DF Exception: Sub: R1063C, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1063C_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-24: DF Exception: Sub: R1093J, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1093J_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-24: DF Exception: Sub: R1093J, Sess: 2, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1093J_2_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-25: DF Exception: Sub: R1100D, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1100D_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-25: DF Exception: Sub: R1100D, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1100D_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-25: DF Exception: Sub: R1100D, Sess: 2, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1100D_2_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-26: DF Exception: Sub: R1120E, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1120E_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-26: DF Exception: Sub: R1122E, Sess: 2, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1122E_2_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-30: DF Exception: Sub: R1154D, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1154D_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-34: DF Exception: Sub: R1186P, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1186P_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-35: DF Exception: Sub: R1201P, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1201P_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-35: DF Exception: Sub: R1201P, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1201P_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-36: DF Exception: Sub: R1216E, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1216E_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-36: DF Exception: Sub: R1216E, Sess: 1, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1216E_1_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-36: DF Exception: Sub: R1216E, Sess: 2, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1216E_2_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-40: DF Exception: Sub: R1277J, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1277J_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
2020-06-15_15-52-57: DF Exception: Sub: R1413D, Sess: 0, FileNotFoundError, [Errno 2] No such file or directory: '/scratch/john/SWR_scratch/SWR_FR1_R1413D_0_HPC_surrounding_recall_soz_in.p', file: <ipython-input-5-866c54e2d1d8>, line no: 25
